src/content.js

```diff
@@ -15,13 +15,25 @@ let links = document.body.getElementsByTagName('a');
 
 // seed the content_delegate with the tabId by using the message 
 // returned from the background worker
-function addRecapInformation(msg){
-    // destructure the msg object to get the tabId
-    const { tabId } = msg;
-    
+function addRecapInformation(msg) {
+  // destructure the msg object to get the tabId
+  const { tabId } = msg;
+
+  if (PACER.isAppellateCourt(court)) {
+
+    const AppDelegate = new AppellateDelegate(tabId, court, url, links);
+
+    if (!PACER.hasPacerCookie(document.cookie)) {
+      console.info(`RECAP: Taking no actions because not logged in: ${url}`);
+      return;
+    }
```

Comment:
Can we move this outside of this if statement as well as out of the else statement below?